addons/mail/static/src/components/follower_subtype_list/follower_subtype_list.js

```diff
@@ -57,11 +44,11 @@ export class FollowerSubtypeList extends Component {
 }
 
 Object.assign(FollowerSubtypeList, {
-    components,
     props: {
         localId: String,
     },
     template: 'mail.FollowerSubtypeList',
 });
 
+registerMessagingComponent(FollowerSubtypeList);
 QWeb.registerComponent('FollowerSubtypeList', FollowerSubtypeList);
```

Comment:
wtf, I didn't even know that was there, we actually need a global register here?